modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/util/TaxonomyCategoryBriefUtil.java

```diff
@@ -48,6 +52,33 @@ public static TaxonomyCategoryBrief toTaxonomyCategoryBrief(
 		};
 	}
 
+	private static Scope _getScope(AssetCategory assetCategory) {
+		Group group = GroupLocalServiceUtil.fetchGroup(
+			assetCategory.getGroupId());
+
+		if (group == null) {
+			return null;
+		}
+
+		return new Scope() {
+			{
+				setExternalReferenceCode(group::getExternalReferenceCode);
+				setType(() -> _getScopeType(group));
+			}
+		};
+	}
+
+	private static Scope.Type _getScopeType(Group group) {
+		if (group.isCMS()) {
+			return Scope.Type.CMS;
```

Comment:
In the story acceptance criteria, Beni mentions `Space` instead of `CMS`. Is this the expected one?

Reply:
@ccorreagg please see the discussion with the BPM team, referenced in the ticket